tests/imbu/unit/imbu_models_test.py

```diff
@@ -311,56 +323,62 @@ def testMergeRanges(self):
     """ Tests the mergeRanges() method used in fragmenting Imbu results."""
     imbu = self._setupFakeImbuModelsInstance()
 
-    def mergeIntoList(ranges):
-      return list(imbu._mergeRanges(ranges))
-
-    ranges = [(3, 7), (3, 5), (0, 4)]
-    self.assertItemsEqual([(0, 7)], mergeIntoList(ranges))
-    ranges = [(5, 6), (3, 4), (1, 2)]
-    self.assertItemsEqual([(1, 2), (3, 4), (5, 6)], mergeIntoList(ranges))
-    ranges = [(0, 13)]
-    self.assertItemsEqual(ranges, mergeIntoList(ranges))
-    ranges = [(0, 3), (6, 3)]
+    ranges = [[3, 7], [3, 5], [0, 4]]
+    self.assertItemsEqual([[0, 7]], list(imbu._mergeRanges(ranges)))
```

Comment:
Consider using `assertSequenceEqual()`.  Although it doesn't apply in this specific case, `assertItemsEqual()` is agnostic to order.